Configure.cmake

```diff
@@ -2,12 +2,9 @@ include(CheckCCompilerFlag)
 include(CheckCSourceCompiles)
 include(CheckTypeSize)
 
-# Some toolchains require explicit linking of the libraries following.
-find_library(LIB_MPFR mpfr)
-find_library(LIBM m)
-find_library(LIBGMP gmp)
-find_library(LIBRT rt)
-find_library(LIBFFTW3 fftw3)
+# Detect if cmake is running on CI environments
+string(COMPARE NOTEQUAL "" "$ENV{TRAVIS}" RUNNING_ON_TRAVIS)
+string(COMPARE NOTEQUAL "" "$ENV{APPVEYOR}" RUNNING_ON_APPVEYOR)
```

Comment:
Please make these variables to be set at `cmake` invocation, as in `cmake -DRUNNING_ON_TRAVIS=On ...`. It is safer than having to deal with things breaking in the case the name of the env variable change.

Reply:
Done.